tools/ci/launch_cluster.sh

```diff
@@ -15,7 +17,7 @@ while [ x"${LAUNCH_SUCCESS}" == x"False" ]; do
     else
         set -e
     fi
-    /venvs/wrap.sh dcos-launch dcos-launch wait --info-path=${CLUSTER_INFO_FILE} 2>&1 | tee dcos-launch-wait-output.stdout
+    dcos-launch wait --info-path=${CLUSTER_INFO_FILE} 2>&1 | tee dcos-launch-wait-output.stdout
```

Comment:
we probably want to change this soon? (not in this PR)

Reply:
aac94c0f5ea0477534192715aaa42b9de5258822